error-handler.js

```diff
@@ -98,15 +99,16 @@ var mixIn = require('mout/object/mixIn'),
     stream.pipe(res);
   },
 
-  send = function send(statusCode, err, res, o) {
+  send = function send(statusCode, err, res, o) {    
+    if (!err.message) err.message = statusCodes[statusCode];
```

Comment:
Ok, but that doesn't explain why you're mutating the original error object... Would something like this work?

```js
send = function send(statusCode, err, res, o) {    
  var base = {
    name: err.name,
    status: statusCode
  };

  if (!err.message) base.message = statusCodes[statusCode];

  var body = (o.serializer) ?
    o.serializer(mixIn({}, base, err)) :
    base;

  res.status(statusCode);
  res.send(body);
}
```